src/org/labkey/test/components/ui/entities/EntityInsertPanel.java

```diff
@@ -397,7 +370,7 @@ public EntityInsertPanel showFileUpload()
             var newPanel = new EntityInsertPanel.EntityInsertPanelFinder(getDriver()).findWhenNeeded(getDriver());
             
             newPanel.clearElementCache();
-            newPanel.waitForLoaded();
+            newPanel.waitForReady();
```

Comment:
Huh, I'd arrived at the need to override waitForReady on EntityInsertPanel separately, in another PR.
We might want to coordinate on that to avoid merge conflicts?

Reply:
Oh, yeah. I forgot about this. I'll pull in your change to customize the wait time. Your `waitForReady` is missing the check for `targetEntityTypeSelect().isInteractive()` from the old `waitForLoaded`.